src/Speckle.Sdk/Serialisation/BaseObjectDeserializerV2.cs

```diff
@@ -140,29 +141,34 @@ public Base Deserialize(string rootObjectJson)
     }
   }
 
-  private object? DeserializeTransportObjectProxy(string? objectJson)
+  private object? DeserializeTransportObjectProxy(string? objectJson, long? current, long? total)
```

Comment:
Only thing I'd raise is that it is not immediately obvious what `current` and `total` are supposed to be used for, nor the fact that both are required (as far as I can gather) if you want the progress to be computed right.

Reply:
renaming for clarity